tests/test.rs

```diff
@@ -1,3 +1,5 @@
+use std::process::Child;
```

Comment:
I would put this import in `lib.rs`, since it's common to import `std::process::Child` as just `Child`.

Reply:
Done.